jetbrains-core/src/software/aws/toolkits/jetbrains/core/executables/ExecutableCommon.kt

```diff
@@ -65,6 +65,30 @@ class ExecutableCommon {
             }
         }
 
+        @JvmStatic
+        fun checkSemVerVersionForParallelValidVersions(
+            version: SemVer,
+            versionRange: List<ExecutableVersionRange>,
+            executableName: String
+        ) {
+            // for use of AWS CLI version 1 and version 2 in ECS Exec
```

Comment:
Don't think we need this comment, especially since it is in common code and not related to that feature specifically

Reply:
done